### PR TITLE
BUG: Bumped sklearn version requirement by a micro level.

### PR DESCRIPTION
**Pull request by kcrum**
Ensure the sklearn version is >= 0.18.1, per the bug in issue #3.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-scikit-learn~=0.18
+scikit-learn>=0.18.1
```
Comment (beckermr) [inline requirements.txt]: We should prob use this

```
scikit-learn~=0.18,>=0.18.1
```

so that we don't bump a version.

Comment (beckermr) [reply requirements.txt]: OTOH, we could just require sklearn 0.19.

Comment (kcrum) [reply requirements.txt]: Hmmm. Would `scikit-learn~=0.18,>=0.18.1` prohibit 0.19? I'd like to be as permissive as possible, and things seem to work with any release >= 0.18.1 (including 0.19). What do you think?

Comment (beckermr) [reply requirements.txt]: Maybe `scikit-learn>=0.18.1,<0.20`?

**Comment by kcrum** [comment]
I just pushed `scikit-learn>=0.18.1,<0.20` to the requirements.txt, but if you think it'd be better to change the numpy deps, we could do that instead. I'm indifferent.

**Comment by beckermr** [comment]
This seems good to me. LGTM!

**Pull request merged**